tests/go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/snapcore/snapd/tests
```

Comment:
why this one?

Reply:
Erm. This was not intended to be changed and I don't know what caused it, will remove.